app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardViewModel.kt

```diff
@@ -24,12 +24,14 @@ class DashboardViewModel(
     private val exposureNotificationManager: ExposureNotificationManager
 ) : ScopedViewModel(appDispatchers) {
 
-    private var wasExposureFrameworkAutomaticallyEnabledOnFirstStart: Boolean
+    var wasExposureFrameworkAutomaticallyEnabledOnFirstStart: Boolean
         get() = dashboardRepository.exposureFrameworkEnabledOnFirstStart
         set(value) {
             dashboardRepository.exposureFrameworkEnabledOnFirstStart = value
         }
 
+    val shouldDisplayWhatsNew = changelogManager.shouldDisplayChangelog
```

Comment:
Should this be a getter to reflect changes in `changelogManager.shouldDisplayChangelog` which is dynamic?

Reply:
No this is intentional, see https://github.com/austrianredcross/stopp-corona-android/pull/188#discussion_r447491329

Reply:
OK. Got it :-)